### ISSUE TITLE
Change to using Add-WindowsPackage or DISM instead of wusa.exe (install slowness and uninstall problems)

### ISSUE THREAD

**Issue by mmundy1** [chocolatey/choco#1989]
WUSA is slower in some cases than Add-WindowsPackage (or possibly DISM if pre-Windows 2016) - For reference, see https://dennisspan.com/slow-installation-of-msu-files-using-the-wusa-exe/

Also, choco uninstall of an MSU removes the PACKAGE but not the actual hotfix in WinServ2016 (confirmed), and win10 (anecdotal) since wusa.exe /quiet does not function correctly.  See https://social.technet.microsoft.com/Forums/Lync/en-US/f6594e00-2400-4276-85a1-fb06485b53e6/issues-with-wusaexe-and-windows-10-enterprise?forum=win10itprogeneral for reference

The latter is a bug.  Should this ticket be separated into Enhancement (install) and Bug (uninstall)?

**Comment by ferventcoder** [comment]
Yes, separated please. Two different issues. However one to think about - some hotfixes can not be uninstalled. While you could remove the chocolatey package all day long, the actual software not so much.

**Comment by mmundy1** [comment]
Breaking out the removal as a bug and leaving the install as an enhancement.  Understood that some packages can't be removed.  However, wusa.exe when not running in quiet mode did remove the package (but of course can't work with win_chocolatey because of the GUI popup that follows)
